Package.swift

```diff
@@ -170,9 +170,12 @@ let package = Package(
 
         // MARK: SwiftPM specific support libraries
 
+        .systemLibrary(name: "SPMSQLite3", pkgConfig: "sqlite3"),
```

Comment:
sorry about the back and forth. maybe we should call this `SystemSQLite3`? that would be more accurate?